modules/cli/src/main/scala/scala/cli/commands/util/FmtOptionsUtil.scala

```diff
@@ -12,7 +12,7 @@ object FmtOptionsUtil {
       val osArchSuffix0 = osArchSuffix.map(_.trim).filter(_.nonEmpty)
         .getOrElse(FetchExternalBinary.platformSuffix())
       val tag0           = scalafmtTag.getOrElse("v" + version)
-      val gitHubOrgName0 = scalafmtGithubOrgName.getOrElse("alexarchambault/scalafmt-native-image")
+      val gitHubOrgName0 = scalafmtGithubOrgName.getOrElse("virtuslab/scalafmt-native-image")
```

Comment:
I think we should still use the former URL, for versions lower than `3.5.9`. Unless these versions' artifacts can be copied over to the new repository too?

Reply:
I didn't find any way, how to copy artifacts from old repository to new

Reply:
I guess you'd need to write a script to do it (using the GitHub API to create releases and upload things to the new repo).